app/helpers/orchestration_stack_helper/textual_summary.rb

```diff
@@ -56,11 +56,12 @@ def textual_ems_cloud
   def textual_orchestration_template
     template = @record.orchestration_template
     return nil if template.nil?
+    return nil unless template.orderable
```

Comment:
@mzazrivec i think we don't need this check anymore, we do want to show a link for both types of Orchestartion Templates from Stack summary screen,